WORKING_MODE.md

```diff
@@ -0,0 +1,12 @@
+# Working mode of the AMP TSC
+
+- We meet every 2 weeks for 1h via video conference.
+- We meet in person 2x per year (timed around the major AMP conferences "AMP Conf" (Spring) and "AMP Contributor Summit" (Fall)).
+- Meeting notes are published in [this repository]((https://github.com/ampproject/meta-tsc/labels/TSC%20Meeting)).
```

Comment:
nit: link format is broken a bit with double parens?

Reply:
Done